composer.json

```diff
@@ -16,6 +16,7 @@
     },
     "require": {
         "php": "^7.4|^8.0",
+        "symfony/polyfill-php80": "^1.22",
```

Comment:
Why polyfill?
`Stringable` used only in phpdoc and use. All work in PHP7.4 without polyfill.